src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -123,6 +124,35 @@ function GeneralizationControls({
   }
 }
 
+function NullsInAidWarning(schema: TableSchema, aidColumn: string) {
+  const computedResult = useNullAid(schema, aidColumn);
+
+  switch (computedResult.state) {
+    case 'completed': {
+      return computedResult.value ? (
+        <>
+          <Alert
```

Comment:
The `<>` is needed if you have more than 1 child. For a single child you can remove it.

Also, this can be converted from ternary operator to `computedResult.value && <Alert ... />` because `false` values are ignored by react.

Reply:
for the latter part: actually, if I do that and https://github.com/diffix/desktop/pull/245/files#r743555212, then it complains about the `false` being returned in lieu of the component. It accepts `Element | null` only